packages/compass-indexes/src/components/property-column/property-column.jsx

```diff
@@ -24,113 +42,78 @@ class PropertyColumn extends PureComponent {
+              >
+                <Icon glyph="InfoWithCircle" />
+              </Link>
+            </Badge>
+          </span>
+        )}
+      >
+        <Body>{tooltip}</Body>
+      </Tooltip>
     );
   }
 
-  /**
-   * Render cardinality info.
-   *
-   * @returns {React.Component} The cardianlity info.
-   */
   renderCardinality() {
-    if (this.props.index.cardinality === 'compound') {
-      return (
-        <div
-          className={classnames(styles['property-column-property-cardinality'])}
-        >
-          {this.props.index.cardinality}
-          <InfoSprinkle
-            helpLink={getIndexHelpLink('COMPOUND')}
-            onClickHandler={this.props.openLink}
-          />
-        </div>
-      );
+    const { cardinality } = this.props.index;
+    if (cardinality !== 'compound') {
+      return null;
     }
+    return this.renderItemWithTooltip(
+      cardinality,
+      getIndexHelpLink('COMPOUND')
+    );
   }
 
-  /**
-   * Render the property column
-   *
-   * @param {String} prop - The property.
-   *
-   * @returns {React.Component} The property component.
-   */
   renderProperty(prop) {
-    const tooltipOptions = {
-      'data-for': TOOLTIP_ID,
-      'data-effect': 'solid',
-      'data-border': true,
-    };
+    const tooltip =
+      prop === 'ttl'
+        ? this._ttlTooltip()
+        : prop === 'partial'
+        ? this._partialTooltip()
+        : null;
 
-    if (prop === 'ttl') {
-      tooltipOptions['data-tip'] = this._ttlTooltip();
-      return (
-        <div
-          {...tooltipOptions}
-          key={prop}
-          className={classnames(styles['property-column-property'])}
-        >
-          {prop}
-          <InfoSprinkle
-            helpLink={getIndexHelpLink('TTL')}
-            onClickHandler={shell.openExternal}
-          />
-        </div>
-      );
-    } else if (prop === 'partial') {
-      tooltipOptions['data-tip'] = this._partialTooltip();
-      return (
-        <div
-          {...tooltipOptions}
-          key={prop}
-          className={classnames(styles['property-column-property'])}
-        >
-          {prop}
-          <InfoSprinkle
-            helpLink={getIndexHelpLink('PARTIAL')}
-            onClickHandler={shell.openExternal}
-          />
-        </div>
-      );
-    }
-    return (
-      <div
-        key={prop}
-        className={classnames(styles['property-column-property'])}
-      >
-        {prop}
-        <InfoSprinkle
-          helpLink={getIndexHelpLink(prop.toUpperCase())}
-          onClickHandler={shell.openExternal}
-        />
-      </div>
+    return this.renderItemWithTooltip(
+      prop,
+      getIndexHelpLink(prop.toUpperCase()),
+      tooltip
     );
   }
 
-  /**
-   * Render the property column.
-   *
-   * @returns {React.Component} The property column.
-   */
   render() {
-    const properties = map(this.props.index.properties, (prop) => {
-      return this.renderProperty(prop);
-    });
+    const properties = map(
```

Comment:
Maybe not the best time for this so feel free to ignore, but we might be able to remove lodash map here and use default js.
```suggestion
    const properties = this.props.index.properties?.map(
```
Looks like lodash map is used in a few other parts of this package so we can't remove it as a dependency just yet, but reducing usage will get us there eventually.